calculator.c

```diff
@@ -0,0 +1,141 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+int numberStack[50];
+int nSTop = -1;
+int operatorStack[50];
+int opSTop = -1;
+
+int evaluateExp(char exp[]);
+void calculate();
+int operatorPrecedence(char operator);
+int custom_isdigit(char ch);
+int custom_isspace(char ch);
+
+int main()
+{
+    char exp[80];
+    int result = 0;
+    fgets(exp, sizeof(exp), stdin);
```

Comment:
what will happen if I enter string greater than 80 characters?

Reply:
fgets will read only the first 79 characters and ignore the rest. which might lead to unexpected results.